### PR TITLE
Added a feature to send an email upon the detection of an intruder

### PR DESCRIPTION
**Pull request by KaranS09**

### CODE REVIEW

File: Main/mail.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+
+# name the email subject
+subject = "Intruder Alert"
+
+
+
+# Define the email function (dont call it email!)
+def send_emails(path):
+    email_list = ["treemusketeers32@gmail.com"]
```
Comment (Hardvan) [inline Main/mail.py]: You can add the college email ids of our team members as well here.

**Review by Hardvan — CHANGES_REQUESTED** [review]
Awesome! I have added a comment to include the list of emails of our team members as recipients of the mail. You can make a commit and incorporate the changes. GG.

File: Main/mail.py (hunk 1 above)
Comment (KaranS09) [reply Main/mail.py]: cool

File: Main/mail.py [hunk 2]
```diff
@@ -29,7 +29,7 @@
 
 # Define the email function (dont call it email!)
 def send_emails(path):
-    email_list = ["treemusketeers32@gmail.com"]
+    email_list = ["treemusketeers32@gmail.com", "harshitdhoot.cs21@rvce.edu.in", "hardikhpawar.cs21@rvce.edu.in", "abhishekyadav.cs21@rvce.edu.in "]
```
Comment (Hardvan) [inline Main/mail.py:32]: Looks good!

**Review by Hardvan — APPROVED** [review]

**Pull request merged**